bindings/node/.eslintrc.json

```diff
@@ -58,9 +71,9 @@
   },
   "overrides": [
     {
-      // Settings for javascript test files
+      // Settings for test files
       "files": [
-        "test/**/*.js"
+        "test/**/*.ts"
```

Comment:
since the test directory is going to be from scratch, let's just remove these overrides, both of the disabled rules are valid for tests just not the tests we had present when we introduced the config